tests/benchmarks/FCSBenchmarks/CompilerServiceBenchmarks/ValueOptionBenchmarks.fs

```diff
@@ -0,0 +1,56 @@
+﻿namespace FSharp.Compiler.Benchmarks
```

Comment:
What value does it serve to the reader to keep this specific file around in a compiler repository?

I had the same feeling with most of other 'test plain F# usage'-benchmarks kept in this compiler repo, but this one raises my confusion the most.

Reply:
The benchmarks are here to provide at least some proof of the PR's value. And for reviewers to check that I am benchmarking things correctly.

As opposed to some other benchmarks in the repo, I don't expect these ones to be used in the near future, so I don't deem them necessary. OTOH I don't expect them to break.

If you wish, I can transform them to a GH gist.

Reply:
IMO it's fine just to delete them, neither repo nor gist needed for that sort of thing.

Reply:
Removed